app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
```suggestion
    user = User.find_by(email: params.dig(:session, :email)&.downcase)
```
Truy cập bằng dig + toán thử an toàn để tránh crash app, tương tự cho password